akirachix/Upile-Informational
# Feature/footer

# Engineers Checklist

Please include a summary of the change and which issue is fixed. 
This PR introduces a footer section for the website. The footer will provide essential  information to enhance user experience and accessibility

Please also include relevant motivation and context. 
The addition of the footer is essential for offering users convenient access to importan information, such as contact details.

List any dependencies that are required for this change.
Tailwind
Next.js
Design

## What this PR does?
* What is this?
This PR implements the footer section of the informational website, including links to important pages and contact information.

* Why this?
Adding a footer will enhance user experience and provide a more complete interface.

* How are you doing this? 
 Created a new footer layout based on design specifications and added the information needed.
Ensured the footer is responsive and properly displayed on various screen sizes.

- List of updates for features, enhancements, or bugfixes
Created footer layout
Applied styling according to the design.
Added responsiveness to the footer.

## Type of change

Specify the kind of PR/MR
New Feature

Please delete options that are not relevant.


## How can it be tested?
Open the infomational website and scroll to the bottom to view the new footer
Test  the footer on different devices to confirm the footer displays correctly across various screen sizes.
![Screenshot from 2024-09-17 17-55-02](https://github.com/user-attachments/assets/ef316c2e-bad9-4882-95f8-8a90941d5766)


# Where is it documented?
- List JIRA tickets
- https://wambuinatalygakenye.atlassian.net/browse/UP-120?atlOrigin=eyJpIjoiYTk3MWQwM2U3MTRkNDQ5MjgzOGY4NWNkY2FiZDE0NmYiLCJwIjoiaiJ9
